scripts/circle_deploy.sh

```diff
@@ -14,49 +13,14 @@ git config --global push.default simple
 # generate changelog
 #
 echo "...generating changelog"
-curl -X POST github-changelog-api.herokuapp.com/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME \
- | json contents > CHANGELOG.md
-
-git add CHANGELOG.md
-
-if [[ -n $(git status --porcelain) ]]; then
-  echo "...starting push, CHANGELOG.md is dirty after build"
-  git commit -n -m "deploy CHANGELOG.md by $CIRCLE_USERNAME [ci skip]"
-  git push origin $CIRCLE_BRANCH
-else
-  echo "...skipping push, repo is clean after build"
-fi
+ta-script circle_ci/create_changelog.sh
```

Comment:
All this logic is now handled by ta-scripts.  Also, it has been updated to a more proper approach.